pyintegration/deephaven2/plot/selectable_dataset.py

```diff
@@ -24,3 +29,51 @@ def __init__(self, j_sds: jpy.JType):
     @property
     def j_object(self) -> jpy.JType:
         return self.j_sds
+
+
+def one_click(t: Table, by: List[str] = None, require_all_filters_to_display: bool = False) -> SelectableDataSet:
+    """ Creates a SelectableDataSet with the specified columns from a table.
+
+    Args:
+        t (Table): the source table
+        by (List[str]): the selected columns
+        require_all_filters_to_display (bool): false to display data when not all oneclicks are selected; true to only
+            display data when appropriate oneclicks are selected
+
+    Returns:
+        a SelectableDataSet
+
+    Raises:
+        DHError
+    """
+    if not by:
+        by = []
+    try:
+        return SelectableDataSet(j_sds=_JSelectables.oneClick(t.j_table, require_all_filters_to_display, *by))
+    except Exception as e:
+        raise DHError(e, "failed in one_click.") from e
+
+
+def one_click_table_map(tm: jpy.JType, t: Table, by: List[str] = None,
```

Comment:
Should we just combine all of these into a `one_click` method?  That seems more pythonic.

Reply:
Yeah  definitely doable when the Java API doesn't require both TableMap and Table in the same method.

Reply:
most likely this will be changed after Ryan completes his rewrite of TableMap.